osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-bundle.xsd

```diff
@@ -495,6 +503,13 @@
       </xsd:complexContent>
     </xsd:complexType>
   </xsd:element>
+  <xsd:element name="DeCoupleMbusDeviceByChannelResponse">
+    <xsd:complexType>
+      <xsd:complexContent>
+        <xsd:extension base="sminst:DeCoupleMbusDeviceByChannelResponseData"></xsd:extension>
```

Comment:
`<xsd:extension base="sminst:DeCoupleMbusDeviceByChannelResponseData"/>` would be more consistent.

Reply:
Removed this and similar inconsistenties from
 - sm-bundle.xsd
 - sm-installation.xsd
 - sm-monitoring.xsd